### PR TITLE
FIX update GEri general description

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

**Comment by manucarrace** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ For documentation previous to Orion 0.23.0 please check the manuals at FIWARE pu
...
+Orion Context Broker allows you to manage all the whole lifecycle of context information including updates, queries, registrations and subscriptions. It is an NGSIv2 server implementation to manage context information and its availability. Using the Orion Context Broker, you are able to create context elements and manage them through updates and queries. In addition, you can subscribe to context information so when some condition occurs (e.g. the context elements have changed) you receive a notification. These usage scenarios and the Orion Context Broker features are described in this documentation.
```
[the remark this answers is not in the document]
Comment (fgalan) [reply README.md]: Fixed in 0524cf724ec3a599a8f3ebbea7ae7a8d81d2abcb

**Comment by fgalan** [comment]
I have take this PR also to remove some reference to contextBroker-test packake (nobody uses it and it is probably very obsolete and unmantained).

**Comment by kzangeli** [comment]
LGTM

**Comment by fgalan** [comment]
I'm also using this PR to fix the formatting of tittles in some .md files.

**Comment by jmcanterafonseca** [comment]
LGTM

**Pull request merged**